devops/deploy-on-lightsail.yml

```diff
@@ -39,13 +39,8 @@
         state: restarted
         daemon_reload: yes
 
-    - name: update pipenv dependencies
-      shell: pipenv update
-      args:
-        chdir: /home/ubuntu/new-train-tracker 
-
-    - name: update npm
-      shell: npm update
+    - name: update app dependencies
+      shell: npm install
```

Comment:
@mathcolo why the change from from `update` to `install`?

Reply:
`npm update` actually changes package.json, which is useful as an action I might take manually, but is not something deploy should be doing

For that matter `npm ci` is actually probably the most appropriate command here! (https://docs.npmjs.com/cli/v6/commands/npm-ci)

Reply:
@mathcolo sounds good, i'd be open to trying it out! perhaps it could be used in our other CD definitions as well if it works here